app/api/resources/mentorship_relation.py

```diff
@@ -315,9 +315,11 @@ def get(cls, request_id):
             return marshal(response, list_tasks_response_body), 200
 
 
-@mentorship_relation_ns.route('mentorship_relation/<int:request_id>/task/<int:task_id>/complete')
+@mentorship_relation_ns.route('mentorship_relation/<int:request_id>/task/<int:task_id>')
 class UpdateTask(Resource):
 
+    auth_header_parser.add_argument('is_done', type=inputs.boolean, help="Whether task should be done or not.")
```

Comment:
Could you please put this parameter as part of the request body instead of as a query parameter?

Reply:
But why?
This would require changing tests and, most importantly, mobile apps' server communication logic.

Reply:
Checking your tests, you call the service as `PUT /mentorship_relation/%s/task/%s?is_done=true`

In REST APIs I usually see query parameters being used for filtering purposes. This is not about querying data, but about updating data. Also, you are adding that `is_done` to  `auth_header_parser` which serves only for the Header (as mentioned in the variable name). There are other ways using Flask RESTplus to add query parameter requirements.
What do you mean with changing mobile apps' server communication logic?

Can I get a second opinion here please? @annabauza @ramitsawhney27 @m-murad

Reply:
Okay, this makes sense.
Regarding mobile's app, it would require moving `is_done` to body instead of params, but that should be fast&easy.

I'll try to complete this PR in the upcoming days.

Reply:
I thought it would be quick and easy :) but we never had a `PUT .../complete` with an is_done, so where would you need to change? 🤔  Glad it makes sense :) I asked for second opinions to untie this. If you feel certain about an approach you should continue the discussion about it :)